acados/ocp_nlp/ocp_nlp_cost_nls.c

```diff
@@ -68,6 +68,8 @@ void ocp_nlp_cost_nls_dims_initialize(void *config_, void *dims_, int nx, int nu
     dims->ny = ny;
     dims->ns = ns;
 
+    // TODO nz
```

Comment:
Why aren't we setting nz?

Reply:
Well, you implemented algebraic variables only for linear least squares, no?

Reply:
For nonlinear least squares, it is TODO